packages/modal/addon/utils/event-handler.js

```diff
@@ -0,0 +1,15 @@
+import { isPresent } from '@ember/utils';
+
+export function bindEvent(context, eventName, callback, element) {
+  let eventCallback = callback.bind(context);
+  let domElement = isPresent(element) ? document.querySelector(element) : document;
```

Comment:
What happens when element is passed with a selector thats not present in the DOM? 

Also, I feel there are too many args to this function. Can this be refactored to unpack fields from an object passed as function parameter? Something like,

```js
function bindEvent({ context = this, eventName, callback, element = document } = {})
```
This way, setting a function parameter's default value is simpler, and it's easier for a future developer to read the function's signature.